installscripts/wizard/scenarios/support/jazz_sonar.py

```diff
@@ -70,32 +68,27 @@ def get_add_existing_sonar_config(terraform_folder):
             "Kindly provide an 'Admin' Sonar user with correct password and run the installer again!"
         )
 
-    # Get Sonar public ip
-    sonar_server_public_ip = raw_input("Sonar Server PublicIp :")
-
     # Create paramter list
     parameter_list = [
-        sonar_server_elb, sonar_username, sonar_passwd,
-        sonar_server_public_ip
+        sonar_server_elb, sonar_username, sonar_passwd
     ]
 
     add_sonar_config_to_files(parameter_list)
 
+
 def get_and_add_docker_sonar_config(sonar_docker_path):
     """
         Launch a dockerized Sonar server.
     """
-    os.chdir(sonar_docker_path)
-    print("Running docker launch script")
-    subprocess.call([
-        'sg', 'docker', './launch_sonar_docker.sh', '|', 'tee', '-a',
-        '../../docker_creation.out'
-    ])
+    encrypt_passwd = hashlib.md5()
+    encrypt_passwd.update(str(datetime.datetime.now()))
+    sonar_passwd = encrypt_passwd.hexdigest()
+    sonar_server_elb = str(ec2_metadata.public_ipv4)
+
     # Get values to create the array
-    parameter_list = []
-    with open("docker_sonar_vars") as f:
-        for line in f:
-            parameter_list.append(line.rstrip())
+    parameter_list = [
```

Comment:
You may still be working on this part, but...if you've only got one value, you don't have a list, and you don't need an array.